ParabolInc/parabol
# fix: fix recreating invite link in case of expiration

fix #9062

I'm not exactly sure why #9062 happens.
We show dots if token is invalid, then we are fetching another token from the frontend.
I may suggest that on mobile with slow connection user may see dots for a while while we are fetching a new token.

However, this should rarely happen because if token is expired, we create new on the backend before returning it. 

This mechanism was broken. I fixed it, which possibly fixes the whole issue

**How to test:**
- Create a fresh team
- Open invite dialog, copy invitation ID from the URL for example XSDBR1S0TF0X
- Expire the url by running:

```
r.db('actionDevelopment')
  .table("MassInvitation")
 .get('XSDBR1S0TF0X')
 .update({
   expiration: new Date()
 })
```

- Refresh page, open invite dialog again, see that a new invite URL appears and you didn't see dots while it was loading
